pkg/apis/dmi/v1beta1/api.proto

```diff
@@ -228,7 +228,8 @@ message CustomizedValue {
 message PushMethod {
     PushMethodHTTP http = 1;
     PushMethodMQTT mqtt = 2;
-    DBMethod dbMethod = 3;
+    PushMethodOTEL otel = 3;
+    DBMethod dbMethod = 4;
```

Comment:
Changed the message number, there will be compatibility issue with older version.
ptal @WillardHu @Shelley-BaoYue @wbc6080 @JiaweiGithub 
/hold